collectors.rb

```diff
@@ -90,23 +88,26 @@ def collect_postgres(host, user, password)
   end
 end
 
-# These are hard-coded for now, but will be moved to a config file soon.
-clusters = ['ganeti']
-db_creds = [{:type => :mysql,
-             :host => 'testing-mysql',
-             :user => 'root',
-             :password => 'toor' }]
 
 c = Collectors.new
 
+# To test the collect_ganeti method, uncomment the following lines.
```

Comment:
Actually that's not really the intention. I can make it more clear but this isn't for *unit tests* but it's more for manual *how does this code work?* testing. I'll clarify the comment but it's something we have a lot of in the codebase.